crypto/codec/amino_test.go

```diff
@@ -0,0 +1,18 @@
+package codec
+
+import "os"
+
+// nolint: govet
+func ExamplePrintRegisteredTypes() {
+	amino.PrintTypes(os.Stdout)
+	// Output:
+	// | Type | Name | Prefix | Length | Notes |
+	// | ---- | ---- | ------ | ----- | ------ |
+	// | PubKey | tendermint/PubKeySr25519 | 0x0DFB1005 | variable |  |
```

Comment:
Please note that I explicitly removed amino support for PrivKeyLedgerSecp256k1. I cannot see where it's needed, but if it's needed, I can add back amino support (in the same way we do for other pubkeys).